src/modelbench/benchmark_runner.py

```diff
@@ -322,15 +252,23 @@ def __init__(self, test_run: TestRunBase):
 
     def handle_item(self, item: TestRunItem):
         for sut in self.test_run.suts:
-            self.downstream_put(TestRunItem(item.test, item.test_item, sut))
+            run_item = TestRunItem(item.test, item.test_item, sut)
+            self.test_run.journal.item_entry(
```

Comment:
I think its also important to record the SUT options

Reply:
The SUT options are the same the whole way through, right? If so, I'd like to put them in once per SUT. What's the best way to get them?

Reply:
>  The SUT options are the same the whole way through, right?

In practice, yes. Each prompt has it's own SUT options, so technically sut options could vary by prompt. But all of safe tests for 0.5 and 1.0 use the same options for all of their respective test items, so it's probably fine to record them once. 

> I'd like to put them in once per SUT.

The option values are provided by the test, so I think it should be recorded once per test instead. 

> What's the best way to get them?

Maybe from a test's first test item via `test_item.prompts[0].prompt.options`

Reply:
And is there an easy way to say what files were used?

Reply:
You could maybe do `dependency_helper.versions_used()` which maps the dependency key to a has of the file contents.
```
"dependency_versions": {
        "all_practice_skilled_unskilled": "fd103633bbfe747a2a90e9daee9d49ace3064733d43ace70fbac88a71b3827d0"
    }
```

It's also *possible* to get the actual source URL, but it's not simple to get. If we want the source URL recorded as well, I think we should defer that to another PR.

Reply:
Also yeah that looks good! Thanks.

Reply:
I rummaged in the debugger and this turned out not to be too bad. In `ModelgaugeTestWrapper` I've been accumulating bits of code that serve to bridge what ModelBench wants a test to be and what ModelGauge thinks of it. It was pretty easy to add there. So now this record looks like this:

```
{
  "timestamp": "2024-10-25T16:13:57.513810+00:00",
  "message": "test info",
  "class": "BenchmarkRunner",
  "method": "run",
  "test": "safe-sxc-en_us-1.0",
  "initialization": {
    "module": "modelgauge.tests.safe_v1",
    "class_name": "SafeTestVersion1",
    "args": [
      "safe-sxc-en_us-1.0",
      "sxc",
      "en_US",
      [
        "normal",
        "skilled",
        "unskilled"
      ]
    ],
    "kwargs": {}
  },
  "sut_options": {
    "max_tokens": 500,
    "temperature": 0.01
  },
  "dependencies": {
    "all_practice_skilled_unskilled": {
      "source": "https://ai-safety-dev.uc.r.appspot.com/files/download/all_practice_skilled_unskilled.csv",
      "local_path": "run/tests/all_practice_skilled_unskilled/fd103633bbfe747a2a90e9daee9d49ace3064733d43ace70fbac88a71b3827d0",
      "timestamp": "2024-10-25T15:26:18.073360+00:00"
    }
  }
}
```

I included the timestamp of the local file so that we can catch issues of updating something on the server but not locally.